.github/workflows/phpunit.yml

```diff
@@ -68,7 +74,8 @@ jobs:
           apt-get install -y postgresql-client
           PGHOST=postgres PGUSER=drupal PGPASSWORD=drupal psql --command='CREATE EXTENSION pg_trgm;' drupal
       - name: Run PHPUnit Action
-        uses: discoverygarden/phpunit-action@v1
+        #uses: discoverygarden/phpunit-action@v1
+        uses: discoverygarden/phpunit-action@adjust-matrix
```

Comment:
Remember to adjust before merging

Reply:
Adjusted back to the major version tag v1